src/test/java/seedu/address/model/person/PhoneTest.java

```diff
@@ -27,14 +27,21 @@ public void isValidPhone() {
         // invalid phone numbers
         assertFalse(Phone.isValidPhone("")); // empty string
         assertFalse(Phone.isValidPhone(" ")); // spaces only
-        assertFalse(Phone.isValidPhone("91")); // less than 3 numbers
         assertFalse(Phone.isValidPhone("phone")); // non-numeric
         assertFalse(Phone.isValidPhone("9011p041")); // alphabets within digits
-        assertFalse(Phone.isValidPhone("9312 1534")); // spaces within digits
+        assertFalse(Phone.isValidPhone("(+9999) 93121534")); // with country code too long
+        assertFalse(Phone.isValidPhone("(+) 93121534")); // with blank country code
+        assertFalse(Phone.isValidPhone("93  121534")); // with more than 1 space
+        assertFalse(Phone.isValidPhone("93121534 [Office And Others]")); // with note > 10 chars
 
         // valid phone numbers
+        assertTrue(Phone.isValidPhone("91")); // short phone number
```

Comment:
One minor thing, I'm not sure if there exists phone numbers that are less than 3 digits? Since the initial requirement was at least 3 digits. Otherwise LGTM!

Reply:
It's in the same spirit of making it more flexible! As for real use cases: sometimes companies/firms may have internal phone numbers that are quite short. 

Thanks for your inputs!